admin_guide/metrics_recommendations.adoc

```diff
@@ -0,0 +1,30 @@
+== OpenShift v3.5 metrics recommendations
+
+In OpenShift v3.4 metrics https://docs.openshift.org/latest/install_config/cluster_metrics.html[documentation]
+are presented results which changed to better in OpenShift v3.5 metrics
+
+The reason for better results lies in change of `METRIC_RESOLUTION` parameter
+in OpenShift metrics v3.5. New value for `METRICS_RESOLUTION` is 30 (seconds) and
+that change was introduced in order to better match cAdvisor housekeeping
+interval of 30 seconds. More details can be found in https://bugzilla.redhat.com/show_bug.cgi?id=1421834[Bugzilla]
+
+Increasing `METRICS_RESOLUTION` interval in OpenShift v3.5 helped to get better results in relation
+how many pods is possible to monitor by one set of metrics pods. Tests showed that OpenShift metrics
```

Comment:
"In OpenShift v3.5, tests showed that OpenShift metrics collection was stable for test cases when up to 25000"